src/index.html

```diff
@@ -16,17 +16,32 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="body">
+    <form
+      class="search-bar big"
+      data-qa="big"
+    >
+      <svg class="icon__big"></svg>
```

Comment:
Consider using a semantic tag like <button> for the icon if it is meant to be interactive or a part of form submission.